packages/aws-cdk-lib/aws-iam/README.md

```diff
@@ -369,6 +369,37 @@ new iam.Role(this, 'Role', {
 });
 ```
 
+### Granting assume role permission from a role
```

Comment:
"From a role" is confusing here. Can you change this to "Granting principals permission to assume a role"